convertAngle.ts

```diff
@@ -0,0 +1,45 @@
+type Angle = "degree" | "radian";
+
+/**
+ * Converts angle values from one unit to another.
+ * Supported units: degrees, radians
+ *
+ * @example
+ * ```
+ * convertAngle(180, "degree", "radian") // 3.141592653589793
+ * convertAngle(MAth.PI, "radian", "degree") // 180
+ * ```
+ */
+const convertAngle = (value: number, from: Angle, to: Angle): number => {
```

Comment:
Add
```ts
assert(
  oneOf(from, "degree", "radian"),
  `convertAngle: unknown unit ${from}`,
  TypeError,
);
```
before the `from === to` check. Same assertion for `to`. That way the function will throw the type error even when `from` and `to` have the same (invalid) value. Then you can also remove the `default:` case from the switch in `convertAngle` and `toDegree`. `assert` is exported from `except.ts` and `oneOf` is one of the existing utils.

Reply:
Did it, using AssertionError